source/develop/release-management/features/virt/engine-dedicated-cpus.md

```diff
@@ -0,0 +1,102 @@
+---
+Title: Engine Dedicated CPUs
+Category: feature
+Authors: sabusale
+---
+
+# Engine Dedicated CPUs
+
+## Summary
+
+Once choosing dedicated CPU policy, The guest vCPUs will be strictly pinned to a set of host pCPUs
+(similarly to static CPU pinning). The set of pCPUs will be chosen to match the required guest CPU topology.
+
+Engine handles the pinning automatically when starting the VM as opposed to the manual pinning that is
+done by the user and is static.
+
+## Owner
+
+-   Name: Saif Abu Saleh (sabusale)
+-   Email: sabusale@redhat.com
+
+## Responsibilities
+
+VDSM:
+
+- Manages the available pinned CPUs
```

Comment:
what does it mean `Manages the available pinned CPUs`?
I'd write it differently, when looking at the engine and vdsm parts in the process of running vms with dedicated cpus then vdsm makes sure that exclusive pinning is preserved by modifying other vms that are not set with dedicated cpus
for example, let's say that I have a VM running on the host with vCPU X that runs on pCPU 1 and the engine decides to pin vCPU Y of another VM with dedicated CPUs to pCPU 1, then the engine specifies that Y should be pinned exclusively to pCPU 1 and VDSM makes sure that X will not run on pCPU 1

Reply:
Changed the wording